Dockerfile

```diff
@@ -2,20 +2,23 @@
 
 # STEP 1: Build kindnetd binary
 FROM --platform=$BUILDPLATFORM golang:1.23 AS builder
-ARG TARGETARCH BUILDPLATFORM
+ARG TARGETARCH BUILDPLATFORM TARGETPLATFORM
 # copy in sources
 WORKDIR /src
 COPY . .
 # build
 RUN CGO_ENABLED=0 GOARCH=$TARGETARCH go build -o /go/bin/kindnetd ./cmd/kindnetd
 WORKDIR /src/cmd/cni-kindnet
 # sqlite requires CGO
-RUN if [ "$TARGETARCH" = "arm64" ] ; then \
-      apt-get update && apt-get -y install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu ;\
-      CC=aarch64-linux-gnu-gcc CGO_ENABLED=1 GOARCH=$TARGETARCH go build -o /go/bin/cni-kindnet . ;\
-  else \
-      CGO_ENABLED=1 GOARCH=$TARGETARCH go build -o /go/bin/cni-kindnet . ;\
-  fi
+# RUN if [ "$TARGETARCH" = "arm64" ] ; then \
```

Comment:
did you comment this on purpose?

Reply:
By mistake, was for a local test. Will fix in a few hours.

Reply:
no worries, I incorporated it in https://github.com/aojea/kindnet/pull/126  , I want to trigger a new build to see if I get the CI working today